content/en/real_user_monitoring/guide/monitor-browser-rum-data-for-electronjs-apps.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Monitor RUM Data for Electron JS Apps 
+kind: guide
+beta: true
+description: Learn how monitor desktop applications that use Electron.js through the browser RUM SDK
+aliases:
+- /real_user_monitoring/guide/getting-started-feature-flags/
+further_reading:
+  - link: '/real_user_monitoring/'
+    tag: 'Documentation'
+    text: 'Learn about Real User Monitoring'
+---
+
+<div class="alert alert-warning">
+    Electron.js support for RUM is in beta.
```

Comment:
When referring to the framework, we see: 
- Electron
- Electron.js
- Electron JS
On their official site: https://www.electronjs.org/ (which is called electronjs), they seem to use only the "Electron" naming (without JS). I am wondering if it wouldn't be better to use only one of the three namings for our article. I think I'd go with just "Electron".